packages/broker/test/integration/plugins/storage/StorageNode.test.ts

```diff
@@ -8,7 +8,6 @@ import {
 import { Broker } from "../../../../src/broker"
 import StreamrClient from 'streamr-client'
 
-const httpPort1 = 12501
 const trackerPort = 12503
 
 describe('StorageNode', () => {
```

Comment:
We could convert this test to a unit test (or remove, as it actually tests the logic of `StreamrClient` node id calculation from a private key)